client/js/screens/EventCategory/EventCategory.js

```diff
@@ -0,0 +1,111 @@
+import React, {useState} from 'react';
+import {Text, Animated, StatusBar, FlatList, View} from 'react-native';
+import {VibrancyView} from '@react-native-community/blur';
+import CardEvent from '../../components/CardEvent';
+import SortActionSheet from '../../components/SortActionSheet';
+import ButtonFilter from '../../components/ButtonFilter';
+import styles from './styles';
+import {THEME} from '../../config';
+import PropTypes from 'prop-types';
+
+const AnimatedFlatList = Animated.createAnimatedComponent(FlatList);
+
+const EventCategory = ({eventInfo, collapsible, navigation, onSwitchTheme}) => {
+  const [isAnimated, setAnimated] = useState(false);
+  const [headerAnimation, setHeaderAnimation] = useState(
+    new Animated.ValueXY({x: 0, y: -88}),
+  );
+  const [showActionSheet, toggleActionSheet] = useState(false);
+  const [sortOption, setSortOption] = useState(null);
+  const [isRating, setRating] = useState(false);
+  const [isPetFriendly, setPetFriendly] = useState(false);
+
+  const {paddingHeight, animatedY} = collapsible;
+
+  const onScroll = e => {
+    const offset = e.nativeEvent.contentOffset;
+    if (offset.y > 88 && !isAnimated) {
+      setAnimated(true);
+      animateHeader();
+      onSwitchTheme(isAnimated);
+    } else if (offset.y <= 88 && isAnimated) {
+      setAnimated(false);
+      animateHeader();
+      onSwitchTheme(isAnimated);
+    }
+  };
+
+  const animateHeader = () => {
+    Animated.timing(headerAnimation, {
+      duration: 500,
+      toValue: isAnimated ? {x: 0, y: -88} : {x: 1, y: 0},
+    }).start();
+  };
+  //   console.log(eventInfo);
+  return (
+    <>
+      {!paddingHeight && <View style={{height: 88}} />}
```

Comment:
Please define what is 88, -88. Also, remove commented code.

Reply:
Defined and removed